openrouteservice/src/test/java/org/heigit/ors/fastisochrones/partitioning/InertialFlowTest.java

```diff
@@ -98,12 +98,12 @@ public void testDisconnect() {
         }
         threadPool.shutdown();
         //Check for partitioning. Cell numbers are not too relevant.
-        int cellId0 = nodeToCell[0];
-        int cellId1 = nodeToCell[3];
-        int cellId2 = nodeToCell[6];
+        Integer cellId0 = nodeToCell[0];
+        Integer cellId1 = nodeToCell[3];
+        Integer cellId2 = nodeToCell[6];
         assertArrayEquals(new int[]{cellId0, cellId0, cellId0, cellId1, cellId1, cellId0, cellId2, cellId2, cellId2, cellId2, cellId1, cellId1}, nodeToCell);
-        assertFalse(cellId0 == cellId1);
-        assertFalse(cellId1 == cellId2);
-        assertFalse(cellId2 == cellId0);
+        assertNotNull(cellId1);
+        assertNotNull(cellId2);
+        assertNotNull(cellId0);
```

Comment:
nodeToCell is an primitive in array, so boxing the entries to Integer makes no sense in this test. What was the reason for the change?